README.md

```diff
@@ -35,7 +37,7 @@ with tfe.protocol.Pond(server0, server1, crypto_producer) as prot:
     result_op = prot.define_output([result], result_receiver)
 
     with config.session() as sess:
-        tfe.run(sess, tf.global_variables_initializer())            
+        tfe.run(sess, tf.global_variables_initializer())
```

Comment:
not sure you need that -- did you run into trouble?

Reply:
This was just my editor removing trailing whitespace

Reply:
ah, sorry! i thought it was the whole line 👍